docs/i18n-languages/spanish/README.md

```diff
@@ -6,7 +6,7 @@
 
 ## ¡Bienvenido al plan de estudios y repositorio público de freeCodeCamp.org!
 
-[freeCodeCamp.org](https://www.freecodecamp.org) es una comunidad amigable donde puedes aprender a programar gratis. Es administrado por una organización sin fines de lucro respaldada por donantes ([donor-supported 501(c)(3) nonprofit](https://donate.freecodecamp.org)) y cuyo objetivo es ayudar a millones de adultos en la transición laboral a campos vinculados con la tecnología. Nuestra comunidad ya ha ayudado a más de 10,000 de personas a obtener su primer trabajo como desarrolladores.
+[freeCodeCamp.org](https://www.freecodecamp.org) es una organización sin fines de lucro que consta de una plataforma web de aprendizaje interactiva, un foro comunitario en línea, salas de chat, publicaciones en línea y organizaciones locales que tienen la intención de hacer que el desarrollo web de aprendizaje sea accesible para cualquier persona. Nuestra comunidad ya ha ayudado a más de 10,000 de personas a obtener su primer trabajo como desarrolladores.
```

Comment:
- "sin ánimo de lucro" should be the term as defined in RAE: https://dej.rae.es/lema/sin-%C3%A1nimo-de-lucro
- "hacer que el aprendizaje de desarrollo web" order reversed in aprendizaje and desarrollo web